### ISSUE TITLE
Report empty lines within lists

### ISSUE THREAD

**Issue by thompson-tomo** [DavidAnson/markdownlint#1958]
## What did you do?

I had a markdown list like the below

```markdown
- a1
- a2

- b1
- b2
```

## What did you expect to happen?

I would like this rogue blank line within the list to have been reported and I had been able to fix all instances of this issue via cli.

**Comment by AlbertoFabbri93** [comment]
I have a similar situation with lists like this one:

```md
1. this

2. is

3. an

4. example
```

that I would like to be reported.

[Tight lists are not mandated by commonmark](https://spec.commonmark.org/0.31.2/#lists) though, so this rule should be disabled by default.
